### ISSUE TITLE
[Feature]: Allow "Execute SQL" tab to be undocked

### ISSUE THREAD

**Issue by Poldraunic** [sqlitebrowser/sqlitebrowser#2985]
### Describe the new feature

I often find myself needing to write an query, but I also do not exactly remember the table schema. And so I keep tabbing in and out between `Browse Data` and `Execute SQL`. It takes quite a lot of time and is kinda unnecessary IMO. It would be nice if I could use it side by side or even in separate window. We already have this functionality with `SQL Log`, `Plot`, `DB Schema`, `Edit Database Cell` and `Remote` widgets.

**Comment by mgrojo** [comment]
In the nightly build you can detach the data browser, so I guess this would cover your use case, because the data browser can be in a detached dock and the Execute SQL tab in the main window.

**Comment by Poldraunic** [comment]
Oh, I didn't know nightly builds existed. Yes this does solve problem a little bit. However, I think it still can be improved upon.

**Comment by chrisjlocke** [comment]
The nightly builds are on the downloads page for both Windows and Mac.  They're here.
https://nightlies.sqlitebrowser.org/latest/

The docks were put into the browse data area first, more of a 'proof of concept' and if it worked without problems, then it would spread across the rest of the application.  Obviously this has been delayed, but this is still the plan.

**Comment by Poldraunic** [comment]
I see. I take it right now its not under active development? If not, I am willing to work on it in my free time.

**Comment by chrisjlocke** [comment]
Db4S is still under active development, but with 500+ issues, there is a bit of a backlog.
This part of DB4s hasn't been implemented past the browse data tab, and I don't know the thought process behind it, eg, how easy it is to implement.  You're of course welcome to branch/fork or whatever and work on it yourself - that's the beauty of open-source.  If you get anything working, you're more than welcome to hand it back and it can be assimilated into our code.  Any assistance in development would be appreciated - someone has to hack away at the 500+ issues! 😁

**Comment by Poldraunic** [comment]
I meant active development of this specific feature.

What I am basically asking is there a plan or somewhat defined end goal as to how this UI should look like. Simply put, I wouldn't want to spend time if my solution won't align with earlier (current) implementation and/or its goals.

**Comment by chrisjlocke** [comment]
No, not of this feature.  There was an idea of how it could be implemented and thus it was tested against the browse data pane, which arguably is the most popular tab in Db4S, or at least, could benefit most from multiple tabs.
The whole DB4S project is a mash of everyone's ideas.  You'll code something and someone later will mould and develop it further.  
All I can suggest is that if you'd like to spend time developing, that time won't be wasted - we won't simply refuse your submission, but equally, don't be upset if its re-written at some point in the future! ;)

**Comment by Poldraunic** [comment]
I see. Well, I will hack something together in couple of weeks time. Thank you.
